feature/ngram_features.py

```diff
@@ -19,4 +19,4 @@ def extractFeatures(self, df):
       counts = self.count_vectorizer.transform(df["comment"])
       features = self.tfidf_transformer.transform(counts)
 
-    return features.tocsr().toarray()
```

Comment:
Leads to the creation of a dense matrix, which leads to a memory error for larger datasets.